mender-convert-modify

```diff
@@ -141,6 +141,22 @@ if [ "${MENDER_ADDON_CONNECT_INSTALL}" = "y" ]; then
         work/rootfs/etc/systemd/system/multi-user.target.wants/mender-connect.service"
 fi
 
+if [ "${MENDER_ADDON_CONFIGURE_INSTALL}" = "y" ]; then
+
+  log_info "Installing Mender Configure addon"
+
+  if [ "${MENDER_ADDON_CONFIGURE_VERSION}" = "latest" ]; then
+    deb_name=$(deb_from_repo_dist_get "work/deb-packages" ${MENDER_APT_REPO_URL} ${deb_arch} "stable" "mender-configure")
+  elif [ "${MENDER_ADDON_CONFIGURE_VERSION}" = "master" ]; then
+    deb_name=$(deb_from_repo_dist_get "work/deb-packages" ${MENDER_APT_REPO_URL} ${deb_arch} "experimental" "mender-configure")
+  else
+    DEBIAN_REVISION="-1"
+    deb_name=$(deb_from_repo_pool_get "work/deb-packages" ${MENDER_APT_REPO_URL} ${deb_arch} "mender-configure" "${MENDER_ADDON_CONFIGURE_VERSION}${DEBIAN_REVISION}")
+  fi
+
```

Comment:
Note to self:

Missing creating `/data/mender-configure` dir and symlink `/var/lib/mender-configure`.

Reply:
Done